src/panels/lovelace/editor/view-editor/hui-edit-view.ts

```diff
@@ -272,6 +291,12 @@ export class HuiEditView extends LitElement {
     }
   }
 
+  private _viewVisibilityChanged(ev: ViewVisibilityChangeEvent): void {
```

Comment:
```suggestion
  private _viewVisibilityChanged(ev: HASSDomEvent<ViewVisibilityChangeEvent>): void {
```

Reply:
After changing this code looks like this:
```
  private _viewVisibilityChanged(
    ev: HASSDomEvent<ViewVisibilityChangeEvent>
  ): void {
    if (ev.detail.detail.visible && this._config) {
      this._config.visible = ev.detail.detail.visible;
    }
  }
```
I must use `detail` twice. Is that ok?

Reply:
Ok, I got this :) Didn't saw the previous comment

Reply:
done